features/support/env.rb

```diff
@@ -5,6 +5,17 @@
 require 'capybara/rails'
 require 'capybara/poltergeist'
 require 'aruba/cucumber'
+require 'database_cleaner/cucumber'
+
+begin
+  DatabaseCleaner.strategy = :truncation
+rescue NameError
+  raise "You need to add database_cleaner to your Gemfile (in the :test group) if you wish to use it."
```

Comment:
Is this line needed if we include the database cleaner on the test group?

Reply:
Not really, I believe it was included as a 'best practice' but happy to remove it if you feel it's redundant

Reply:
Thanks for the review, removed 👍